roles/postgres/tasks/setup_db.yml

```diff
@@ -0,0 +1,63 @@
+---
+- name: Include OS specific variables
+  ansible.builtin.include_vars: "{{ item }}"
+  loop: >-
+    {{ lookup('first_found', os_fallback, errors='ignore', wantlist=True) }}
+
+- name: Configure postgresql client auth
+  become: true
+  notify:
+    - Restart-postgresql
+  ansible.builtin.blockinfile:
+    path: "{{ postgresql_conf_path }}/pg_hba.conf"
+    block: |
+      {% for host in postgres_db_clients | map('extract', hostvars, ['ansible_default_ipv4', 'address']) %}
+      host {{ postgres_db_name }} {{ postgres_db_username }} {{ host }}/32 md5
```

Comment:
ok if we want to enforce the /32 here maybe better to make it clear in the arguments description

Reply:
the argument desc already mentions a list of inventory hosts is expected so it would always be /32